root/static/scripts/edit/MB/Control/ArtistCredit.js

```diff
@@ -76,7 +76,9 @@ MB.Control.ArtistCreditName = aclass(MB.entity.ArtistCreditName, {
 
         } else if (/^&$/.test(join)) {
             this.joinPhrase(" & ");
-        }
+        }else if(/^;$/.test(join) ){
```

Comment:
Minor, but you could change the spaces here so that they are like the others (before "else", after "if", after "join)". Similarly, you seem to have more spaces than the rest of the file - it's generally best to make everything have the same style, so please take a look at changing that :)